browser/ui/webui/brave_settings_ui.cc

```diff
@@ -139,6 +140,11 @@ void BraveSettingsUI::AddResources(content::WebUIDataSource* html_source,
     }
   }
 
+  for (size_t i = 0; i < kEmailAliasesGeneratedSize; ++i) {
+    html_source->AddResourcePath(UNSAFE_TODO(kEmailAliasesGenerated[i]).path,
```

Comment:
I don't think we should be adding new usages of `UNSAFE_TODO` cc @cdesouza-chromium

Reply:
As mentioned in the presubmit:

> Do not use UNSAFE_TODO() to write new unsafe code. Use only when removing a pre-existing file-wide allow_unsafe_buffers pragma, or when incrementally converting code off of unsafe interfaces

This potentially should be something like

```cxx
for (const auto& email_alias : base::span(kEmailAliasesGenerated)) {
  html_source->AddResourcePath(email_alias.path, email_alias.id);
}
```

Reply:
this definitely does not seem right, I only see this pattern being used in `BraveSettingsUI::AddResources` and that probably needs to be updated, but I'm not sure what the correct thing to do here is

Reply:
fixed